docs/content/preview/reference/configuration/yb-tserver.md

```diff
@@ -311,59 +311,52 @@ Default: `64`
+Default: `-1` (the value based on the CPU cores is calculated at runtime).
+\
+For servers with up to two CPU cores, the default value is considered as `4`.
+\
+For three or more CPU cores, the default value is considered as `8`.
+\
+If [`enable_automatic_tablet_splitting`](#enable-automatic-tablet-splitting) is `true`, then the default value is considered as `1` and tables will begin with 1 tablet *per node*; for version `2.18` and later, for servers with up to 4 CPU cores, the value *is not defined* and tables will begin with 1 tablet (for servers with up to 2 CPU cores), or 2 tablets (for servers with up to 4 CPU cores) *per cluster*.
 
-{{< note title="Important" >}}
+**For clusters installed with `yb-ctl`, `yb-docker-ctl`, and `yugabyted`**
 
-This value must match on all `yb-master` and `yb-tserver` configurations of a YugabyteDB cluster.
-
-{{< /note >}}
-
-{{< note title="Important" >}}
-
-When the value is set to *Default* (`-1`), then the server internally *updates* the flag with intended value during startup prior to version `2.18` and the flag remains *unchanged* starting from version `2.18`.
-
-{{< /note >}}
+Local cluster installations created with `yb-ctl` and `yb-docker-ctl` use a default value of `2` for this flag.
+Clusters created with `yugabyted` always use a default value of `1`.
 
 {{< note title="Note" >}}
 
-On a per-table basis, the [`CREATE TABLE ... WITH TABLETS = <num>`](../../../api/ycql/ddl_create_table/#create-a-table-specifying-the-number-of-tablets) clause can be used to override the `yb_num_shards_per_tserver` value.
-
-{{< /note >}}
-
-{{< note title="Note" >}}
-
-If [`enable_automatic_tablet_splitting`](#enable-automatic-tablet-splitting) is `true`: the default value is considered as `1` and tables will begin with 1 tablet *per node*; from version `2.18` for servers with up to 4 CPU cores, the value *is not defined* and tables will begin with 1 tablet (for servers with up to 2 CPU cores) or 2 tablets (for servers with up to 4 CPU cores) *per cluster*.
+- This value must match on all `yb-master` and `yb-tserver` configurations of a YugabyteDB cluster.
+- If the value is set to *Default* (`-1`), then the server internally *updates* the flag with intended value during startup prior to version `2.18` and the flag remains *unchanged* starting from version `2.18`.
+- The [`CREATE TABLE ... WITH TABLETS = <num>`](../../../api/ycql/ddl_create_table/#create-a-table-specifying-the-number-of-tablets) clause can be used on a per-table basis to override the `yb_num_shards_per_tserver` value.
 
 {{< /note >}}
 
 ##### --ysql_num_shards_per_tserver
 
 The number of shards per YB-TServer for each YSQL table when a user table is created.
+If the value is -1, the system automatically determines an appropriate value based on the number of CPU cores; it is determined to 1 if `enable_automatic_tablet_splitting` is set to true.
```

Comment:
I don't think we need this change in short description. There's the same information below.